src/cohpsk_internal.h

```diff
@@ -113,7 +113,7 @@ struct COHPSK {
 void bits_to_qpsk_symbols(COMP tx_symb[][COHPSK_NC*COHPSK_ND], int tx_bits[], int nbits);
 void qpsk_symbols_to_bits(struct COHPSK *coh, float rx_bits[], COMP ct_symb_buf[][COHPSK_NC*COHPSK_ND]);
 void tx_filter_and_upconvert_coh(COMP tx_fdm[], int Nc, const COMP tx_symbols[],
-                                 COMP tx_filter_memory[COHPSK_NC][COHPSK_NSYM],
+                                 COMP tx_filter_memory[COHPSK_NC*ND][COHPSK_NSYM],
```

Comment:
Shouldn't this be COHPSK_NC * COHPSK_ND to match line 114?

Reply:
It should be, as you say. The "ND" appears to be a redundant define, and not as specific as "COHPSK_ND". Probably should track down "ND" everywhere and update it.

Reply:
cohpsk_create() seems to use both and even compare each other in assert().

Reply:
If they're the same value, no point in having both. Otherwise, we need to figure out why they're different and at least document that accordingly.

Reply:
I can't remember why we have both 'ND' and 'COHPSK_ND'.  Hopefully the cests will pick up any issues if we decide to reconcile them